app/models/user.rb

```diff
@@ -69,11 +69,62 @@ def can_upload(s3_path)
 
     # Don't allow any non-Biohub users to upload from czbiohub buckets
     if CZBIOHUB_BUCKET_PREFIXES.any? { |prefix| user_bucket.downcase.starts_with?(prefix) }
-      unless ["czbiohub.org", "ucsf.edu"].include?(email.split("@").last)
+      unless biohub_user?
         return false
       end
     end
 
     true
   end
+
+  def biohub_user?
```

Comment:
The string checking was a little hacky to begin with, so in the future we could lean on 'institution' and backfill it for Biohub/UCSF/CZI

Reply:
Yes, institution seems valuable in the long run. Is it currently required and standardized?

Reply:
it is required on the "interest form" but account creation either happens through 1) Product creating the user at /users/new or 2) many accounts created through the Add User To Project feature which auto-creates and invites users. I think we could update both forms to require institution